src/js/models/v2/awards/BaseAwardRecipient.js

```diff
@@ -18,10 +18,17 @@ const parseBusinessCategories = (data) => (
 
 const parseExecutiveCompensation = (data) => {
     const executiveCompensation = {};
-    for (let i = 1; i < 6; i++) {
-        const name = data[`officer_${i}_name`] || '';
-        const amount = formatMoney(data[`officer_${i}_amount`]) || 0;
-        executiveCompensation[`officer${i}`] = `${name} - ${amount}`;
+    if (data) {
+        for (let i = 1; i < 6; i++) {
+            const name = data[`officer_${i}_name`] || '';
+            const amount = formatMoney(data[`officer_${i}_amount`]) || 0;
+            executiveCompensation[`officer${i}`] = `${name} - ${amount}`;
+        }
+    }
+    else {
```

Comment:
I don't think the properly handles the case where some executive compensation is reported, but not all 6 (ie, 4 are reported, the last two slots should be `--`).